README.md

```diff
@@ -69,19 +69,23 @@ Requirements:
 - [Go](https://golang.org/doc/install) 1.15+ (to build the provider plugin)
 
 ## Testing
-How to run the tests isn't obvious.
 
-First, you need a running instance of the JFrog platform (RT and XR). However, there is no currently supported dockerized, local version. You can ask for an instance to test against in as part of your PR. Alternatively, you can run the file [scripts/run-artifactory.sh](scripts/run-artifactory.sh), which, if have a file in the same directory called `artifactory.lic`, you can start just an artifactory instance. The license is not supplied, but a [30 day trial license can be freely obtained](https://jfrog.com/start-free/#hosted) and will allow local development.
+First, you need a running instance of the JFrog Artifactory. 
+You can ask for an instance to test against it as part of your PR. Alternatively, you can run the file [scripts/run-artifactory.sh](scripts/run-artifactory.sh). 
+The script requires a valid license of a [supported type](https://github.com/jfrog/terraform-provider-artifactory#license-requirements), license should be saved in the file called `artifactory.lic` in the same directory as a script.
+With the script you can start one or two Artifactory instances using docker compose.   
+The license is not supplied, but a [30 day trial license can be freely obtained](https://jfrog.com/start-free/#hosted) and will allow local development. Make sure the license saved as a multi line text file. 
 
-Once you have that done you must set the following properties
-
-Then, you have to set some environment variables as this is how the acceptance tests pick up their config
+Currently, acceptance tests **require access key** and don't support basic authentication and API key. To generate access key, please refer to the [official documentation](https://www.jfrog.com/confluence/display/JFROG/Access+Tokens#AccessTokens-GeneratingAdminTokens) 
+Then, you have to set some environment variables as this is how the acceptance tests pick up their config.
 ```sh
 ARTIFACTORY_URL=http://localhost:8082
 ARTIFACTORY_USERNAME=admin
-ARTIFACTORY_PASSWORD=password
+ARTIFACTORY_ACCESS_TOKEN=<your_access_token>
```

Comment:
Please put a comment about why the username is still needed if the access token is in use

Reply:
it's in row 87

Reply:
Ah... sorry